pkgs/native_assets_cli/lib/src/json_utils.dart

```diff
@@ -43,7 +43,8 @@ extension JsonUtils on Map<String, Object?> {
 
   List<Object?> list(String key) => get<List<Object?>>(key);
   List<Object?>? optionalList(String key) => getOptional<List<Object?>>(key);
-  Map<String, Object?>? optionalMap(String key) =>
+  Map<String, Object?> object(String key) => get<Map<String, Object?>>(key);
```

Comment:
This returns a map, not an object. This is quite confusing to read on the call site. (You probably meant to say the values are object. However, the values are an optional object in the type. And if there's no "valuesAre" in the name, it's still really confusing on the call site.)

`object` -> `map`

And below `optionalObject` -> `optionalMap`.

Reply:
I cannot make `object` -> `map` because `map` is already defined on the  type this is an extension on.

Understand that it's at first sight a little confusing. I used `object` because it actually describe it more precisely what it does: It gets the JSON object (they are called "object"s in json) for the given `key` and this extension is dealing with JSON data.

But if you prefer I can call it `dartMap` or something else (though `map` doesn't work due to collision with `Map.map` / `List.map`)

Reply:
> Understand that it's at first sight a little confusing. I used `object` because it actually describe it more precisely what it does: It gets the JSON object (they are called "object"s in json) for the given `key` and this extension is dealing with JSON data.

Ah, well I think it's nicer to structure the API from a Dart terminology, we're getting a Dart object out.

`dartMap` sgtm, or `map_` or @HosseinYousefi's favorite: `map$`.

Reply:
I feel called out on my secret mission to convert Dart to PHP

Reply:
`map$` it is.

Done.